crm_claim_code/models/crm_claim.py

```diff
@@ -20,12 +20,16 @@ class CrmClaim(models.Model):
     @api.model
     def create(self, vals):
         if vals.get('code', '/') == '/':
-            vals['code'] = self.env['ir.sequence'].get('crm.claim')
+            vals['code'] = self.env['ir.sequence'].next_by_code('crm.claim')
         return super(CrmClaim, self).create(vals)
 
-    @api.one
+    @api.multi
     def copy(self, default=None):
         if default is None:
             default = {}
-        default['code'] = self.env['ir.sequence'].get('crm.claim')
-        return super(CrmClaim, self).copy(default)
+        results = self.env['crm.claim']
```

Comment:
more simple than that: copy is not supposed to be called on several records, it is designed to copy only one record at a time, the code can be reduced to:

```python
@api.multi
def copy(self, default=None):
    self.ensure_one()
    if default is None:
        default = {}
    default.setdefault('code', self.env['ir.sequence'].next_by_code('crm.claim'))
    return super(CrmClaim, record).copy(default)
```